app/javascript/react/screens/App/common/networkKey.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * returns a unique network based on current UI model
+ */
+export const networkKey = network => `${network.clusterId}\\${network.uid_ems}`;
```

Comment:
I really love this idea -- `networkKey`

Reply:
@AparnaKarve i'm out of fresh ideas after this one unfortunately... borderline rocket surgery here 🤣

Reply:
The Network id (LAN id) also seems to work here just fine! 
```js
`${network.id}\\${network.uid_ems}`
```

Really good thinking here -- isolating `networkKey` in a separate file !!